src/server/middleware/twitter.middleware.ts

```diff
@@ -330,7 +331,7 @@ function loadTwitterDataV2(
     ...(request.nextPageToken && { next_token: request.nextPageToken }),
     ...{
       query: `(@${user} OR url:twitter.com/${user}) -from:${user} -is:retweet`,
-      max_results: BATCH_SIZE,
+      max_results: ESSENTIAL_OR_ELEVATED_V2_BATCH_SIZE,
```

Comment:
```suggestion
      max_results: credentials.useEssentialOrElevatedV2 ? ESSENTIAL_OR_ELEVATED_V2_BATCH_SIZE : ENTERPRISE_BATCH_SIZE,
```

I take it Academy Research access also uses `500`, no?